classes/phing/tasks/ext/SymlinkTask.php

```diff
@@ -236,7 +236,36 @@ protected function getMap()
             }
 
             if ($this->isRelative()) {
-                $fromDir = $fs->getDir($this->getProject())->getPath();
+                $endPath = $fs->getDir($this->getProject())->getAbsolutePath();
+                $startPath = $this->getLink();
```

Comment:
@siad007 Thanks for that updated and working with a relative path to the project root is given in the command.